statsmodels/stats/contingency_tables.py

```diff
@@ -0,0 +1,1312 @@
+"""
+Methods for analyzing two-way contingency tables (i.e. frequency
+tables for observed units that are classified with respect to either
+one or two categorical variables).
+
+The main classes are:
+
+  * Table : implements methods that can be applied to any two-way
+  contingency table.
+
+  * SquareTable : implements methods that can be applied to a square
+  two-way contingency table.
+
+  * Table2x2 : implements methods that can be applied to a 2x2
+  contingency table.
+
+  * StratifiedTables : implements methods that can be applied to a
```

Comment:
should `StratifiedTables` be in singular form `StratifiedTable` ?
It's one stratified table?